pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -40,9 +40,10 @@ const (
 )
 
 const (
-	// Annotations for driveCR to perform restart process
-	driveRestartReplacementAnnotationKey   = "drive"
```

Comment:
Don't we need to support this key as deprecated and remove later?

Reply:
add annotation key&value as deprecated